src/sonic-yang-models/yang-models/sonic-telemetry.yang

```diff
@@ -71,6 +71,11 @@ module sonic-telemetry {
                     description "Port gnmi runs on.";
                 }
 
+                leaf save_on_set {
```

Comment:
would you please also update sonic-gnmi.yang?

Reply:
Done